app/assets/javascripts/votes.js

```diff
@@ -0,0 +1,50 @@
+$( "a.upvote i" ).click(function( event ) {
+  event.preventDefault();
+  event.stopPropagation();
+  $(this).toggleClass("gold");
+  var reviewId = $(this).attr("id").match(/review-([0-9]+)/)[1];
+  var downvote = "#downvote-" + reviewId + " i";
+  $(downvote).removeClass("gold");
+
+  $.ajax({
+  type: "POST",
+  url: "/reviews/" + reviewId + "/votes",
+  data: { value: 1 },
+  dataType: "json",
+  success: function(){
+    currentVoteCount(reviewId);
+    }
```

Comment:
Indentation is off in lines 10-16. Take a look at an Ajax request in the docs and indent this block appropriately.